test/unit/spec/appWidgets/input/checkboxInputSpec.js

```diff
@@ -54,49 +62,116 @@ define(['widgets/appWidgets2/input/checkboxInput', 'common/runtime', 'testUtil']
             });
         });
 
-        it('should start and stop properly without initial value', () => {
+        it('should start and stop properly without initial value', async () => {
```

Comment:
Some cleanup throughout, mostly moving to the `async/await` style, until the end tests.